### DISCUSSION TITLE
Chapter Zero (Welcome, Certificates, Credits)

### DISCUSSION THREAD

**Discussion by bellabf** [huggingface/computer-vision-course#183] [Q&A]
Hi, everyone.
We came up with the following outline for Unit-0 which will guide the learners how to take the course.

1. **Welcome to the course** ( welcome.mdx)
 - This is where we try to introduce Computer Vision & how its useful, pre-requisites, hardware-requirements, sign-up form, certificates, Credits to the author and a shoutout to the community
 2. **What to expect from this course** (learn.mdx)
  - A firm intro to course, the gradio demo, course structure ( units summary) etc…
  - Chapters to be summed up (brief overview of each unit)
      1.   Unit 1 - Fundamentals
      2.   Unit 2 - Convolutional Neural Networks
      4.   Unit 3 - Vision Transformers
      5.   Unit 4 - Multimodal Models
      6.   Unit 5 - Generative Models
      7.   Unit 6 - Basic CV Tasks
      8.   Unit 7 - Video and Video Processing
      9.   Unit 8 - 3D Vision, Scene Rendering and Reconstruction
      10.   Unit 9 - Model Optimization
      11.   Unit 10 - Synthetic Data Creation
      12.   Unit 11  - Zero Shot Computer Vision
      13.   Unit 12 - Ethics and Biases
      14.   Unit 13 - Outlook

3.  **Set-up** (learn.mdx)
  - About the discord server, how to navigate (maybe), channels about CV on the server

**Questions**: 

1. Is there a certificate being offered to the learner after the completion of this course?
2. What is the best way to give credit to the people who contribute to the course? 
4. How are you planning to access the performance of the learner, we've seen other courses have quizzes and certificates.  Will it be the same for this as well?

Regarding credits, we don't know if the spreadsheet has been kept up to date. I know that at least for our unit (fundamentals/imaging) the spreadsheet was not completely up do to date and that there were contributors that did not upload to the git themselves. @seshu-pavan has started [this spreadsheet](https://docs.google.com/spreadsheets/d/1zo_m-65ohJcddtXxiDskyJfSYfsenjxL2KX4Lxu65HI/edit?usp=sharing) to try to mitigate that.

We look forward to hearing your thoughts :)

cc: @MKhalusova @johko @merveenoyan @lunarflu  @ATaylorAerospace

Best,
Fundamentals Team (@bellabf , @seshu-pavan, @aman06012003 )

**Comment by bellabf** [comment]
I will start the discussion by giving my own opinion on the matter.  Considering that (as far I can see), the course was not designed to have quizzes in my mind.  So, adding quizzes might be a significant amount of effort.

**Comment by ATaylorAerospace** [comment]
> Hi, everyone. We came up with the following outline for Unit-0 which will guide the learners how to take the course.
> 
> 1. **Welcome to the course** ( welcome.mdx)
> 
> * This is where we try to introduce Computer Vision & how its useful, pre-requisites, hardware-requirements, sign-up form, certificates, Credits to the author and a shoutout to the community
> 
> 2. **What to expect from this course** (learn.mdx)
> 
> * A firm intro to course, the gradio demo, course structure ( units summary) etc…
> * Chapters to be summed up (brief overview of each unit)
>   
>   1. Unit 1 - Fundamentals
>   2. Unit 2 - Convolutional Neural Networks
>   3. Unit 3 - Vision Transformers
>   4. Unit 4 - Multimodal Models
>   5. Unit 5 - Generative Models
>   6. Unit 6 - Basic CV Tasks
>   7. Unit 7 - Video and Video Processing
>   8. Unit 8 - 3D Vision, Scene Rendering and Reconstruction
>   9. Unit 9 - Model Optimization
>   10. Unit 10 - Synthetic Data Creation
>   11. Unit 11  - Zero Shot Computer Vision
>   12. Unit 12 - Ethics and Biases
>   13. Unit 13 - Outlook
> 
> 3. **Set-up** (learn.mdx)
> 
> * About the discord server, how to navigate (maybe), channels about CV on the server
> 
> **Questions**:
> 
> 1. Is there a certificate being offered to the learner after the completion of this course?
> 2. What is the best way to give credit to the people who contribute to the course?
> 3. How are you planning to access the performance of the learner, we've seen other courses have quizzes and certificates.  Will it be the same for this as well?
> 
> Regarding credits, we don't know if the spreadsheet has been kept up to date. I know that at least for our unit (fundamentals/imaging) the spreadsheet was not completely up do to date and that there were contributors that did not upload to the git themselves. @seshu-pavan has started [this spreadsheet](https://docs.google.com/spreadsheets/d/1zo_m-65ohJcddtXxiDskyJfSYfsenjxL2KX4Lxu65HI/edit?usp=sharing) to try to mitigate that.
> 
> We look forward to hearing your thoughts :)
> 
> cc: @MKhalusova @johko @merveenoyan @lunarflu @ATaylorAerospace
> 
pppq> Best, Fundamentals Team (@bellabf , @seshu-pavan, @aman06012003 )

Hi @bellabf Thanks for starting off this important conversation. I think if we follow the Hugging Face Audio Course example for course completion and certificates that would be a great format for this course to follow https://huggingface.co/learn/audio-course/chapter8/certification

The audio course makes extensive use of the HF Hub for evaluation etc.

As for course credit that is one of my outstanding questions as well. The Spreadsheet really only has our discord name but not our full name and or contact information that we would want to receive credits to.

**Comment by MKhalusova** [comment]
My 2 cents: 
Re: certification. If you want to create certificates for the learners who complete the course, you need to think about how you will evaluate that they have actually completed the course and understood the material. I highly recommend figuring out an automated way to do so, unless you are prepared to spend hours and hours evaluating submissions manually. For the Audio course, we have designed hands-on exercises where a learner is asked to fine-tune a model and upload it to Hub. This allows them to apply what they have learned to a slightly different data/language/task. This offers a challenge because it's not exactly like the example in the course and requires an understanding of what one is doing. By uploading the models to Hub, we have access to the metrics and can automatically evaluate with some simple scripts in Spaces. We also use Spaces to generate the certificates for those who pass all the assignments. The exercises should be challenging, but realistic. The metric should be achievable on accessible hardware (avoid exercises that require beefy GPUs), and you should test your exercises for that! 

Re: credit to the people who contribute to the course. Why not include a page in the first unit - "About course authors" (or something like this?) where you could list all of the authors with links to their HF (or GitHub) profiles, and a one-paragraph bio.
